conf/mime.types

```diff
@@ -1,7 +1,8 @@
 
 types {
     text/html                                        html htm shtml;
-    text/css                                         css;
+    text/css                                         CSS;
```

Comment:
Why css was changed?

Reply:
Pardon my mistake, used the GitHub editor to switch lines and somehow I changed this. Reverted.

https://github.com/nginx/nginx/pull/338/commits/a6e2470380ffcb95198d616da1e1df0f9e223e43